src/coldcard.ts

```diff
@@ -410,15 +405,16 @@ export class ColdcardExportExtendedPublicKey extends ColdcardMultisigSettingsFil
  *
  */
 export class ColdcardSignMultisigTransaction extends ColdcardInteraction {
-  /**
-   *
-   * @param {object} options - options argument
-   * @param {string} options.network - bitcoin network
-   * @param {array<object>} options.inputs - inputs for the transaction
-   * @param {array<object>} options.outputs - outputs for the transaction
-   * @param {array<string>} options.bip32Paths - BIP32 paths
-   * @param {object} [options.psbt] - PSBT of the transaction to sign, include it or we will generate it
-   */
```

Comment:
I see some jsdoc stuff has been remove, but the ones attached to comments above the definition block also will affect typing.